auth/proofOfPossession_test.go

```diff
@@ -17,13 +18,6 @@ func TestProofOfPossession_parseAndValidateCertificate_edgeCases(t *testing.T) {
 		assert.Equal(t, "there is no x509 client certificate provided", err.Error())
 	})
 
-	t.Run("validateCertificate() fails when no token is given", func(t *testing.T) {
```

Comment:
why are these test cases deleted?

Reply:
Because the parameter token can't be nil anymore.